ios/Helio.xcodeproj/project.pbxproj

```diff
@@ -2117,14 +2127,14 @@
 				CODE_SIGN_ENTITLEMENTS = BetterSocialStaging/BetterSocialStagingRelease.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 2.8.0.115;
+				CURRENT_PROJECT_VERSION = 2.10.105.1001;
 				DEVELOPMENT_TEAM = J22M3M9VYU;
 				EXCLUDED_ARCHS = "";
 				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
 				INFOPLIST_FILE = "$(SRCROOT)/BetterSocialStaging/Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 11.4;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				MARKETING_VERSION = 2.8.0;
+				MARKETING_VERSION = 2.10.105;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, ensure that the updates to `CURRENT_PROJECT_VERSION` and `MARKETING_VERSION` are intentional and accurately represent the new version of your application.
```diff
-				CURRENT_PROJECT_VERSION = 2.8.0.115;
+				CURRENT_PROJECT_VERSION = 2.10.105.1001;
-				MARKETING_VERSION = 2.8.0;
+				MARKETING_VERSION = 2.10.105;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->